### DISCUSSION TITLE
Soap Karavan example

### DISCUSSION THREAD

**Discussion by Luvgupta** [apache/camel-karavan#868] [General]
Hi  - I am new to Apache Karavan. I had been trying to consume SOAP web services using Apache karavan.

Can someone help me with a sample or describe what field needs to be mapped to exposed fields.

**Comment by 2k20dhd** [comment]
Hello,

I would appreciate, as well, a simple example showing how to consume a SOAP web service using Karavan.
Thank you.

**Reply by nballur** [reply]
I need one simple example too, if you guys can post yaml OR link to sample will help...I am on Apache Camel Karavan 4.8.0.

Any kind of help will us a jump start since many of them or new to Apache Karavan.

**Reply by frankhuurman** [reply]
I've tried creating a simple SOAP service with the cxf component on 4.18.0 and 4.18.1 but haven't gotten a successful build or dev container run yet (missing dependencies and whatnot). Seems harder than it should be with Karavan.

**Comment by davsclaus** [comment]
The Kaoto Forage project can make using CXF easier
https://kaotoio.github.io/forage/
